ui/components/app/recovery-phrase-reminder/recovery-phrase-reminder.js

```diff
@@ -38,7 +43,7 @@ export default function RecoveryPhraseReminder({ onConfirm, hasBackedUp }) {
         >
           {t('recoveryPhraseReminderSubText')}
         </Typography>
-        <Box margin={[4, 0, 8, 0]}>
+        <Box marginTop={4} marginBottom={8}>
           <ul className="recovery-phrase-reminder__list">
             <li>
               <Typography
```

Comment:
Updating previous array props for different directions to singular direction props now that array props are reserved for responsive props